File: practices/time_of_day.py
import datetime

current = datetime.datetime.now()
hour = current.hour

print(f"The time is: {current}")
print(f"hour is: {hour}")
if hour < 12:
    print("Good morning!")
elif hour < 18:
    print("Good afternoon!")
else:
    print("Good evening!")

File: c_programming/Practices/time_of_day.c
// LL 6th Time of Day practice


#include <stdio.h>
#include <time.h>

int main() {
    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);

    int hour = timeinfo->tm_hour;

    if (hour >= 5 && hour < 12) {
        printf("Good morning!\n");
    } else if (hour >= 12 && hour < 18) {
        printf("Good afternoon!\n");
    } else {
        printf("Good evening!\n");
    }

    return 0;
}